### DISCUSSION TITLE
Errors all over the place

### DISCUSSION THREAD

**Discussion by juliux32** [BlafKing/sd-civitai-browser-plus#176] [Q&A]
Why I am getting all these errors no matter what I am searching for?
![2024-01-27 13-28-37](https://github.com/BlafKing/sd-civitai-browser-plus/assets/21074960/32a186eb-3f0b-4996-8035-b25b71fd97a2)

**Comment by BlafKing** [comment]
Unfortunately I can't do much with just seeing these errors, please fill out all the relevant information in the [bug report](https://github.com/BlafKing/sd-civitai-browser-plus/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml&title=%5BBug%5D%3A+) so that I can look into your issue!
